components/ArtPieces/ArtPieces.js

```diff
@@ -0,0 +1,24 @@
+import ArtPiecePreview from "../ArtPiecePreview/ArtPiecePreview";
+import styled from "styled-components";
+
+export function ArtPieces({ data }) {
+  return (
+    <StyledUl>
+      {data.map((artPiece) => {
+        return (
+          <li key={artPiece.slug}>
+            {" "}
+            <ArtPiecePreview artPiece={artPiece} />{" "}
```

Comment:
```suggestion
            <ArtPiecePreview image={artPiece.imageSource} title={artPiece.name} artist={artPiece.artist} />
```
It's not wrong but when we read the user story in detail, this would be the asked solution.

Reply:
Done ✅